example_configs/jellyfin.md

```diff
@@ -1,42 +1,47 @@
 # Configuration for Jellyfin
 
-Replace `dc=example,dc=com` with your LLDAP configured domain.
+Replace all instances of `dc=example,dc=com` with your LLDAP configured domain.
 
+## LDAP Server Settings
 ### LDAP Bind User
+Create an ldap user for Jellyfin to run search queries (and optionally reset passwords). For example `jellyfin_bind_user`
 ```
-uid=admin,ou=people,dc=example,dc=com
+uid=jellyfin_bind_user,ou=people,dc=example,dc=com
 ```
 
 ### LDAP Base DN for searches
 ```
 ou=people,dc=example,dc=com
 ```
 
-### LDAP Attributes
-
+## LDAP User Settings
+### LDAP Search Filter
+If you have a `media` group, you can use:
+```
+(memberof=cn=media,ou=groups,dc=example,dc=com)
+```
+Otherwise, just use:
+```
+(uid=*)
+```
+### LDAP Search Attributes
 ```
 uid, mail
 ```
-
-### LDAP Name Attribute
-
+### LDAP Uid Attribute
 ```
 uid
 ```
-
-### User Filter
-
-If you have a `media` group, you can use:
+### LDAP Username Attribute
 ```
-(memberof=cn=media,ou=groups,dc=example,dc=com)
+uid
 ```
-
-Otherwise, just use:
+### LDAP Password Attribute
 ```
-(uid=*)
+userPassword
```

Comment:
Is that the main difference? It's a bit hard to tell given that the fields were reordered. 

Anyway, this field is not, will not and cannot be available through LLDAP by design.

Reply:
I agree it is difficult to see the diff. I recommend just looking at the entire rendered file side by side.

90% of changes is just reordering and adding some `## Headings` to match the Jellyfin UI sections.
The only modification to existing was to change the default bind user from `admin` to `jellyfin_bind_user`.
The only addition was adding the following block:
```markdown
### LDAP Password Attribute
\```
userPassword
\```
```

The reason I added it in the pr is:
- It was the only attribute from jellyfin UI that wasn't listed in `jellyfin.md`
- `userPassword` is the default jellyfin value and it is functioning as expected for me.
- It is mentioned to at the bottom of `jellyfin.md` under the `password change` section:
```markdown
`LDAP Password Attribute` Needs to be set to `userPassword`
```

Reply:
@nitnelave Given that `userPassword` is not available in lldap, did you want me to remove my addition and the last line of the `Password change` section introduced in c440df6 ?

Reply:
The last line is fine: you can change passwords through jellyfin, but you can't read them (which the attribute is about) 

So yes, please remove the `userPassword` section, the rest of the PR looks fine

Reply:
I've removed the requested section. Let me know if anything else needs changing.